app/notify_admin.go

```diff
@@ -26,7 +27,11 @@ func (a *App) SaveAdminNotification(userId string, notifyData *model.NotifyAdmin
 	requiredPlan := notifyData.RequiredPlan
 	trial := notifyData.TrialNotification
 
-	if a.UserAlreadyNotifiedOnRequiredFeature(userId, requiredFeature) {
+	isInstallPluginNotification := strings.HasPrefix(requiredPlan, model.WorkTemplates)
```

Comment:
I understand the motivation here to leverage and reuse as much as possible what we currently have for notify admin. However, it does seem a bit confusing seeing the plugin install request in the `requiredPlan` 

Wondering if it would fit a bit more with the `RequiredFeature` with the thinking of a plugin as a feature you would need? So maybe something like: 
```suggestion
	isInstallPluginNotification := strings.HasPrefix(requiredFeature, model.PluginFeature)
```

Reply:
We can have the `MattermostPaidFeature` type drop the `Paid` portion of it and be just `MattermostFeature` so we can include the Plugin there as a feature. I already see they added a `UpgradeDowngradedWorkspace` "feature" there so I believe this would make more sense